.github/ISSUE_TEMPLATE/4_change_request.md

```diff
@@ -1,5 +1,5 @@
 ---
-name: Change requests
+name: Complain
 about: Is there something you don't like? Let's talk
 labels: change request
```

Comment:
All good here? Feels like the verb is correct 😃

Reply:
👍